index.md

```diff
@@ -28,24 +28,24 @@ States](https://us-rse.org), and other countries.
 
 ### What is a research software engineer?
 
-A growing number of people in academia combine expertise in programming with an
-intricate understanding of research. Although this combination of skills is
+A growing number of people in academia **combine expertise in programming with an
+intricate understanding of research**. Although this combination of skills is
 extremely valuable, RSEs often lack a formal place in the academic
-system: they produce less papers than a researcher, their code is less
+system: they may produce less papers than a researcher, their code is less
 formal than a software developers', and career recognition is
-correspondingly low.  The term "Research Software Engineer" to attempt
+correspondingly low.  The term "Research Software Engineer" is an attempt
```

Comment:
Perhaps we point out that fewer as first author.  and they may contribute to many papers and not appear an them!

Reply:
Good point. I will add it to this PR.